src/uu/tail/src/tail.rs

```diff
@@ -66,8 +67,8 @@ pub mod options {
 
 #[derive(Debug)]
 enum FilterMode {
-    Bytes(usize),
-    Lines(usize, u8), // (number of lines, delimiter)
+    Bytes(u64),
+    Lines(u64, u8), // (number of lines, delimiter)
 }
```

Comment:
@sylvestre this codecov warning has to be wrong right? it's just an enum type change, surely it's being used / covered in tests?

![image](https://user-images.githubusercontent.com/10882062/155199694-dd5934be-bacd-4ae1-a6ca-5f56a0ccc29d.png)

Reply:
![image](https://user-images.githubusercontent.com/11643477/155202153-a1c2f1d8-67b7-43c8-965d-db930c024a05.png)

According to codecov, it wasn't tested before either, which is indeed strange, because it definitely is (for example in `test_bytes_single`)